### DISCUSSION TITLE
Transpiration (suction/blowing) Boundary Condition in SU2

### DISCUSSION THREAD

**Discussion by jnmiranda-aero** [su2code/SU2#2207] [Q&A]
Hi everyone, 

Does the current version of SU2 support a transpiration (suction/blowing) boundary condition? and if so, what cases/tutorials are there for this types of flows? I came across this [paper](https://arc.aiaa.org/doi/pdf/10.2514/6.2019-0695): Shock-Induced Separation Suppression Using CFD-Based Active Flow Control Optimization where shock-induced separation is mitigated using active flow control (via surface suction). That led me to finding this feature ([feature_transpiration] https://github.com/su2code/SU2/tree/feature_transpiration)), but I'm having some difficulty in finding more information about it, where this feature is at compared to the main branch, how to run the cases shown in the paper and how to apply suction boundary conditions in SU2. 

I would really appreciate any info anyone has about this

Thank you!
Juan

--

**Comment by pcarruscag** [comment]
The official su2 versions do not have that feature, if the author of the branch is still active maybe they'll reply here.
